test/sagemaker_tests/pytorch/training/integration/sagemaker/test_smart_sifting.py

```diff
@@ -32,7 +32,10 @@ def validate_or_skip_smart_sifting(ecr_image):
 
 def can_run_smart_sifting(ecr_image):
     _, image_framework_version = get_framework_and_version_from_tag(ecr_image)
-    return Version(image_framework_version) in SpecifierSet("==2.0.1")
+    image_cuda_version = get_cuda_version_from_tag(image_uri=ecr_image)
+    return Version(image_framework_version) in SpecifierSet("==2.0.1") and (
```

Comment:
nit: this probably should be 2.0.* since we use the same dockerfile for patch versions of a framework typically.

Reply:
Made the change. Thanks!